cpusupport/Build/cpusupport-X86-CRC32.c

```diff
@@ -0,0 +1,17 @@
+#include <smmintrin.h>
+
+#include <stdint.h>
+
+int
+main(void)
+{
+	unsigned int state = 0;
+	unsigned char x = 0;
+	uint8_t buf[8] = {0, 0, 0, 0, 0, 0, 0, 0};
+
+	/* Test both the 8-bit and 64-bit data versions. */
+	state = _mm_crc32_u8(state, x);
+
+	_mm_crc32_u64(state, *(unsigned long *)&buf[0]);
```

Comment:
Maybe worth adding `state =` here and making the next line `return (state);` in order to avoid dead-code warnings?